app/_data/docs_nav_ee_0.35-x.yml

```diff
@@ -137,6 +137,7 @@
             - text: OpenID Connect
               url: /developer-portal/configuration/authentication/oidc
             - text: Adding Custom Registration Fields
+              url: /developer-portal/configuration/authentication/adding-registration-fields
```

Comment:
the staging site is probably not updated, but `Adding Custom Registration Fields` is still taking me to the Enterprise landing page. Going to test locally.

Reply:
Yes it hasn't updated to the second commit

Reply:
Works locally, going to approve.